### ISSUE TITLE
Add term for 4-5 toe cutaneous syndactyly

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10325]
**Preferred term label:**
4-5 toe cutaneous syndactyly

**Synonyms**
[similar based on 2-4 toe cutaneous syndactyly HP:0005768]
Soft tissue syndactyly of toes 4,5 - Webbed 4th and 5th toes - Webbed fourth and fifth toes

**Definition (free text, please give PubMed ID)**
A soft tissue continuity in the anteroposterior axis between the toes 4 and 5
PMID: 37352860

**Parent term (use hpo.jax.org/app)**
Cutaneous syndactyly of toes HP:0010621

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
For example: PMID: 37352860

**Your nano-attribution (ORCID)**
A. Zankl 0000-0001-8612-1062, @azankl
L. Wit 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@azankl

Adding new term: 
- ID: HP:6000503
- label: 4-5 toe cutaneous syndactyly
- parents: [Cutaneous syndactyly of toes (HP:0010621)](https://hpo.jax.org/app/browse/term/HP:0010621) | [4-5 toe syndactyly (HP:0004692)](https://hpo.jax.org/app/browse/term/HP:0004692)
- pmid: PMID:37352860
- def: A soft tissue continuity in the anteroposterior axis between the toes 4 and 5.
- comment: 
- orcid: orcid:0000-0001-8612-1062
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10325
- synonyms:
